lib/async-git-prompt.zsh

```diff
@@ -0,0 +1,33 @@
+ASYNC_PROC=0
```

Comment:
Since this is a long-lived global variable, should have a more distinctive name to indicate its scope. Maybe `OMZ_GIT_ASYNC_PROC`?